src/main/resources/avro/bdg.avdl

```diff
@@ -64,6 +64,28 @@ record Contig {
   union { null, int } referenceIndex = null;
 }
 
+record RecordGroupMetadata {
+  /**
+   Record group identifier.
+   */
+  union { null, string } recordGroupName = null;
```

Comment:
> The prefix in all the fields seems redundant.

Ah yes, it is... I'll drop the prefix.

> And this record could just be called RecordGroup, no?

I think `RecordGroupMetadata` is slightly preferable, since this record just stores the metadata. Depending on how much of a stickler for terminology you are, the "record group" would actually be the full group of reads that are from the same sample/flow cell/lane/etc. Thoughts?